### DISCUSSION TITLE
'Cef' does not exist in the current context

### DISCUSSION THREAD

**Discussion by ZakDevOnGitHub** [cefsharp/CefSharp#4962] [General]
I'm creating a web browser powered by Chromium using CefSharp, and
 for some reason, there is an error message saying `'Cef' does not exist in the current context`. I checked multiple times that I was typing in the correct place and it still gives me this error! I'm using the latest version of `CefSharp.Winforms` (129.0.110) and I couldn't find any CefSharp Web browser tutorial anywhere covering this or other versions that are **very** close to the latest one. I don't want to use the versions in the tutorials I found considering the vulnerabilities found in those versions. If anyone can help me with this issue, it would be so helpful. 🙏
![image](https://github.com/user-attachments/assets/f12c2ee2-91cc-4490-a374-ef1b9939f444)
![image](https://github.com/user-attachments/assets/894eecd2-a1d3-41c1-bd5d-6b075b2d4390)

**Comment by mitchcapper** [comment]
If you are just looking for a CefSharp sample the repo includes a decently robust sample application.   The obvious might be not including the right using directive?  Other potentials might be package restore problems if the library is not referenced at all.  If you go to the top of the class and type `using Cef` does it autocomplete with cefsharp namespaces?

**Reply by ZakDevOnGitHub** [reply]
Thanks! I realized I was supposed to install other NuGet plugins other from `Cef.WinForms` which fixed the issue. Everything is now working as usual! 👍
